### DISCUSSION TITLE
Welcome to aphantasia Discussions!

### DISCUSSION THREAD

**Discussion by eps696** [eps696/aphantasia#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rizqisuganda** [comment]
HI, is there any youtube tutorial  video that can explain how to generate picture to video step by step. honestly i really don't know how to code. but i can draw and make 3d modeling. very interested in this project, thankyou!

**Comment by eps696** [comment]
hi, thanks for your interest. couple of comments:
1) working with these notebooks requires zero code knowledge - all code is hidden in the cells for a reason. you only need some basic skills of dealing with google colab platform (which is as easy as google docs).
2) there's no such feature as "picture to video" amongst aphantasia notebooks for the moment; they can take images as references indeed, but the result will be quite far from the source - not like just effect-processing, which is implemented in some other notebooks (of other authors). 
that said, i hope you'll still be able to enjoy this stuff ::]
